chart/keda/README.md

```diff
@@ -45,6 +45,21 @@ $ helm install kedacore/keda-edge --devel --set logLevel=debug --namespace keda
 func kubernetes install --namespace keda
 ```
 
+## Configuration
+
+| Parameter                         | Description                         | Default              |
+|:----------------------------------|:------------------------------------|:---------------------|
+| `image.repository`                | Repository which provides the image | `kedacore/keda`      |
+| `image.tag`                       | Tag of image to use | `lastest`            |
+| `image.pullPolicy`                | Policy to pull image | `Always`            |
+| `replicaCount`                    | Amount of replicas to run | `1`            |
+| `customResourceDefinition.create` | Indication to whether or not to create the custom resource definition | `true`            |
+| `rbac.create`                     | Indication to whether or not to use role-based access control | `true`            |
+| `serviceAccount.create`           | Indication to whether or not to a serivce account should be used | `true`            |
+| `serviceAccount.name`             | Name of the service account to use | ``            |
+| `logLevel`                        | Granularity of logs to use | `info`          |
+| `glogLevel`                       | Granularity of logs to use | `2`            |
```

Comment:
There seems to be overlap in terms of log level, do we need both?

Reply:
Unfortunately yes. One controls our log messages, and one controls other dependencies that are using a different logger. We should try to unify them though.

Reply:
Created #238 to track this